src/main/java/io/quarkus/registry/app/model/ExtensionRelease.java

```diff
@@ -109,11 +109,11 @@ public static Optional<ExtensionRelease> findByGAV(String groupId, String artifa
 
     public static List<ExtensionRelease> findNonPlatformExtensions(String quarkusCore) {
         DefaultArtifactVersion artifactVersion = new DefaultArtifactVersion(quarkusCore);
-        String minimumVersion = artifactVersion.getMajorVersion() + ".0.0.Final";
-        String maximumVersion = (artifactVersion.getMajorVersion() + 1) + ".0.0.Final";
+        String lowerBound = artifactVersion.getMajorVersion() + ".0.0.Alpha1";
```

Comment:
Wouldn't `Alpha` be enough? It'd be equivalent to `Alpha0`. In fact, I believe you could simply use `artifactVersion.getMajorVersion() + ".a".